src/index.js

```diff
@@ -1,9 +1,9 @@
-'use strict';
-
 export function annotate (...services) {
```

Comment:
I'd actually prefer that this file be plain `module.exports = function annotate () {}`. And babel would only be used to transpile the tests so that they can use the decorator syntax. Keeps things simple, no dist files. #2 referenced browserify and my priority for modules is supporting browserify/webpack (et al) users. If the source code is all just ES5 there's no compilation needed.